src/main/scala/akka/cluster/swissborg/package.scala

```diff
@@ -6,7 +6,7 @@ package object swissborg {
    * Helper to work around `wartremover:NonUnitStatements` warts.
    */
   @specialized def discard[A](evalForSideEffectOnly: A): Unit = {
-    val _: A = evalForSideEffectOnly
```

Comment:
I removed it because the casting is useless, since it will be removed after erasure. When adding it back, there is a warning in 2.13 which specifically states that:

```
[warn] .../lithium/src/main/scala/akka/cluster/swissborg/package.scala:9:12: abstract type pattern A is unchecked since it is eliminated by erasure
[warn]     val _: A = evalForSideEffectOnly
```

But I can add it if you feel is needed